egs2/audioset/codec1/conf/scoring/score.yaml

```diff
@@ -0,0 +1,63 @@
+# codec example yaml config
```

Comment:
If you used the exact same configuration setting, you may directly link it to the libritts recipe ones with symlink